src/libs/blueprint/conduit_blueprint_mesh.hpp

```diff
@@ -351,6 +351,17 @@ namespace topology
                                                   conduit::Node &s2dmap,
                                                   conduit::Node &d2smap);
 
+        //---------------------------------------------------------------------
+        void CONDUIT_BLUEPRINT_API map_fields(const conduit::Node &poly_mesh,
```

Comment:
since this only works for mapping to sides, we should aim for a more descriptive name, how about:

`map_fields_to_generated_sides`